tony-core/src/main/java/com/linkedin/tony/util/Utils.java

```diff
@@ -744,7 +744,7 @@ public static void extractResources(String appId) {
             containerEnv.put(envDockerImage, imagePath);
           } catch (NoSuchFieldException e) {
             LOG.error("Field " + Constants.ENV_CONTAINER_TYPE + " or " + Constants.ENV_DOCKER_CONTAINER_IMAGE + " not "
-                          + "found in " + containerRuntimeClass.getName());
+                          + "found in " + containerRuntimeClass.getName() + " or " + dockerRuntimeClass.getName());
```

Comment:
@zuston  thanks for fixing this, we ran into this confusion as well.. could you log the exception e as well ?

Reply:
Ok. ~